src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -53,38 +56,64 @@
      */
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException{
-        Query query = new Query("Comments").addSort("timestamp", SortDirection.DESCENDING);
-        PreparedQuery userComments = datastore.prepare(query);
+        Entity currentDeed = getCurrentDeed();
+        Collection userComments = (Collection) currentDeed.getProperty("Comments");
```

Comment:
Why not just used a `List` instead of a `Collection`?

Reply:
Good idea! I was initially having problems putting Lists into the datastore so I assumed Lists were the issue and immediately switched over to Collection. However, I just tried again and List worked so the issue was something else that I fixed along the way haha. I will switch to List